haas/ext/auth/database.py

```diff
@@ -137,6 +137,12 @@ def authenticate(self):
         else:
             return False
 
+    def get_user(self):
+        if hasattr(local, 'auth') and local.auth:
```

Comment:
@gsilvis If you see the authenticate method. It sets `local.auth = None` before doing anything. So regardless of if the authentication was successful or not, `local` will have an attribute `auth`. 

Therefore when I call `get_user` on the backend the flow is:
* Is there an attribute called `auth` in `local`?
* Yes, access `local.auth.label` - This failed because `local.auth` is None. 

-